proposals/incubation/kubevirt.adoc

```diff
@@ -0,0 +1,164 @@
+
+### SIGs
+
+* [SIG list](https://github.com/kubevirt/community/blob/master/sigs.yaml)
+
+Most of those SIGs are not yet independent committees. The Performance and Scale SIG has its own leadership and contributors, and both Documentation and Storage SIG should soon have the same. The community plan is to, as the community grows, foster dedicated contributors in each SIG area. As well as expanding our contribution pipeline, this will provide more opportunities for new contributors to rise to leadership of the project.
+
+### Maintainers
+
+* https://github.com/kubevirt/community/blob/master/MAINTAINERS.md
+
+### GitHub source code repository
+
+* About 50 contributors measured by multiple L or greater sized contributions
+* 171 official members in the organization
+* 2562 stars
+* 615 forks
+* 96 watched tags
+
+### Communications
+The project utilizes Google Groups as a mailing list where where users have a chance to interact with core developers and discuss general topics. There are currently (2021/06/01) 436 subscribers.  The mailing list typically receives 0-5 new threads per day.
+
+The project holds a video conference every week where users have a chance to interact with core developers, discuss general topics and participate in bug triage. Recently the performance and scale working group started conducting a weekly meeting based on the topic. Both meetings are recorded and posted to the project YouTube channel. Meeting notes are also emailed to the general mailing list to keep the community informed.
+
+The project uses two channels on the CNCF Kubernetes Slack server. #virtualization is used to handle general use topics. The channel hosts over 1000 users. #kubevirt-dev is used for developer oriented topics. The channel hosts 350 users. The channels typically receive 0-5 new threads per day.
+
+Important announcements are relayed via mailing list, website blog and Twitter.
+
+KubeVirt advertises communications channels via https://kubevirt.io/community as well as project [README](https://github.com/kubevirt/kubevirt/blob/master/README.md).
+
+### End Users
+
+Project end users are maintained in the [Adopters](https://github.com/kubevirt/kubevirt/blob/main/ADOPTERS.md) files
+
+Significant users include:
+* NVIDIA
+* SUSE
+* Kubermatic
+* H3C
+* CoreWeave
+* Civo
+* CloudFlare
+* Ateme
+* Cloudbase Solutions
+
+### Integrations
+
+* [oVirt](https://www.ovirt.org) ... Virtual machines can be run in a container using the external provider feature.
+* [Gardener](https://gardener.cloud/blog/2020-10/00/) ...
+* [Kubermatic Virtualization](https://www.kubermatic.com/products/kubevirt/)
+* [SUSE/Rancher Harvester](https://github.com/rancher/harvester/blob/766abd06561b059c1af623aacc4e505db471ceee/deploy/charts/harvester/README.md)
+* [Google Anthos](https://youtu.be/RE0A3kHT3LA?t=126)
+
+### CNCF Sponsored Security Audit
```

Comment:
We should at least take a note if this took place or not.

Reply:
The section is just a place holder and gets filled in when it occurs during incubation.

Reply:
To me this description would exactly fit into this section: "This section is just a place holder and gets filled in when it occurs during incubation."